CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ This change log adheres to standards from [Keep a CHANGELOG](http://keepachangel
 ## Unreleased
 
 ### Fixed
+* [`display-name`]: fix arrow function returning result of function call with JSX arguments being interpreted as component ([#3065][], @danielfinke)
```

Comment:
this should be added at the bottom of the "fixed" list, and you'll need to add a PR link for `[#3065]` (see `[#3061]` as an example)

Reply:
@ljharb done!

> You may also be able to check the "allow edits" box in the RHS of the PR
Nah, I tried looking for this when I first submitted the PR and saw the failure. Doesn't seem to be an option anywhere for me